plotting feature

bmlindor opened the pull request.

bmlindor committed:
Create Orbits.jl

module to plot initial conditions

bmlindor committed:
Update Orbits.jl

bmlindor committed:
Revert "Update Orbits.jl"

This reverts commit 908faa5319a35f123677fc27fe7f0fdf789a834d.

bmlindor committed:
Revert "Create Orbits.jl"

This reverts commit 14ac8af2b33a9b9f2ea35dec1741210091a1827b.

bmlindor committed:
Merge branch 'ericagol:master' into master

bmlindor committed:
Reapply "Create Orbits.jl"

This reverts commit 64bb6e3c13055dbfe86373bf9bf86c9c91f392c0.

bmlindor committed:
Merge branch 'master' of https://github.com/bmlindor/NbodyGradient

bmlindor committed:
Revert "Merge branch 'master' of https://github.com/bmlindor/NbodyGradient"

This reverts commit 8cfddeee1106015269b60c33994c9d27d255f910, reversing
changes made to 50408159cae001d77620413b437aa3fe95073c43.

bmlindor committed:
Reapply "Merge branch 'master' of https://github.com/bmlindor/NbodyGradient"

This reverts commit bc1178ef0cb439e7777d75c67db4dac14af36840.

bmlindor committed:
Update Orbits.jl

bmlindor commented:
do not be alarmed by the 10 (!) commits. I just had some issues managing the fork  as I had to update to the upstream master.

ericagol reviewed (commented):
This looks useful!  I have a couple of initial comments: 1). Can this file be given a different name than Orbits.jl?  Since Orbits.jl is a Julia package, I think this will cause some confusion. 2). Is there a purpose to have the grad=true option?  Since this plots the orbits, I don't think the derivatives are needed.

bmlindor commented:
changed the name to VisualizeNbodyGradient, and removed the grad option

bmlindor committed:
fixes

removed grad option, changed name, added option to build orbits without existing integrator

langfzac commented:
This looks great, thanks @bmlindor ! Question: are you planning to work on this further? If not, I think the best thing to do would be to make the PR to a dev branch, so that Eric and I can make additions, squash commits, etc, easier.
